jinja.py

```diff
@@ -17,13 +17,13 @@
 UBER_PASS = os.environ.get('UBER_PASS') or ""
 
 JOBS = {}
-HOST = '172.23.98.63'
+HOST = '127.0.0.1'
```

Comment:
I think this needs to be changed back to original IP

Reply:
Ah yes! Thanks for catching!